src/main/java/com/codeup/adlister/dao/MySQLAdsDao.java

```diff
@@ -112,6 +115,47 @@ public Ad getAd(long id) {
             throw new RuntimeException("Error finding profile", e);
         }
     }
+
+
+
+    @Override
+    public void deleteAd(Ad ad) {
+        try{
+            PreparedStatement stmt = connection.prepareStatement("UPDATE ads SET is_deleted = ? WHERE id = ?");
+
+
+            stmt.setBoolean(1, true);
+            stmt.setLong(2, ad.getId());
+
+            stmt.executeUpdate();
+
+        } catch (SQLException e) {
+            throw new RuntimeException("Error updating Profile information", e);
+        }
+
+    }
+
+
+
+//    @Override
```

Comment:
This is more an observation. It is always better to remove code than commenting it out. If you ever need this code again you can always get it back from a commit using git.